Formula/chart-testing.rb

```diff
@@ -0,0 +1,29 @@
+class ChartTesting < Formula
+  desc "Testing and linting Helm charts"
+  homepage "https://github.com/helm/chart-testing"
+  url "https://github.com/helm/chart-testing.git",
+      :tag      => "v3.0.0",
+      :revision => "50db473a1e68c605b18d82f019d83ea401542213"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    commit = Utils.safe_popen_read("git", "rev-parse", "HEAD").chomp
+    ldflags = %W[
+      -X github.com/helm/chart-testing/v3/ct/cmd.Version=#{version}
+      -X github.com/helm/chart-testing/v3/ct/cmd.GitCommit=#{commit}
+      -X github.com/helm/chart-testing/v3/ct/cmd.BuildDate=#{Date.today}
+    ]
+    system "go", "build", "-ldflags", ldflags.join(" "), "-trimpath", "-o", bin/"ct", "./ct/main.go"
+  end
+
+  test do
+    assert_match "Lint and test", shell_output("#{bin}/ct --help")
+
+    # Tries to lint the Helm charts in the current directory which fails
+    # because there are no Helm charts in the current directory
```

Comment:
Those aren't very complex. Why not just write a very basic chart to lint?

Reply:
Now I just create a new Helm chart that will be used for testing with `helm create testchart`